lib/index.js

```diff
@@ -89,6 +89,14 @@ exports.register = (server, pluginOptions, next) => {
         config: {
           auth: false,
```

Comment:
it might 'just work' if you set auth to try and then auth.isAuthenticated is populate